app/api/check-auth/route.ts

```diff
@@ -6,28 +6,28 @@ import logger from '@/utils/server/logger';
 const LOGIN = `${process.env.NEXT_PUBLIC_BASE_URL}/login`;
 
 export async function GET() {
-    try {
-        const token = await getAuthTokenServer();
-
-        if (token) {
-            try {
-                const user = await verifyJwtToken(token);
-                if (user) {
-                    return NextResponse.json({ authenticated: true });
-                } else {
-                    logger.error('User verification failed');
-                    return NextResponse.json({ authenticated: false });
-                }
-            } catch (err) {
-                logger.error('Token verification error', { error: err })
-                return NextResponse.redirect(LOGIN);
-            }
+  try {
+    const token = await getAuthTokenServer();
 
+    if (token) {
+      try {
+        const user = await verifyJwtToken(token);
+        if (user) {
+          return NextResponse.json({ authenticated: true });
         } else {
-            return NextResponse.json({ authenticated: false })
+          logger.error('User verification failed');
```

Comment:
All the other calls to `logger.error()` have an object as the first param, and string as the second. This one does not. Is that intentional?